src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = ".NET Core";
```

Comment:
> For example, you could plausibly say that you are running ".NET Core" assemblies on the desktop framework (we certainly support running a lot of the assemblies in this repo on the desktop framework).

Since this is a OOB assembly, and as u mentioned about nuget packaging, there will be a different implementation assembly for netcore50, net46, dnxcore50 which will be authored internally in TFS with different FrameworkDescription strings. And on referencing the assembly, the Nuget package manager will pick the right implementation depending on the platform. This is just like ensuring you don't run a linux OOB assembly on windows, similarly we ensure we don't run a dnxcore50 library on netcore50.